apps/etterlatte-saksbehandling-ui/client/src/shared/types/Brev.ts

```diff
@@ -61,3 +62,12 @@ export enum Spraak {
   NN = 'nn',
   EN = 'en',
 }
+
+export enum Brevtype {
+  VEDTAK = 'VEDTAK',
+  VARSEL = 'VARSEL',
+  INFORMASJON = 'INFORMASJON',
+  OPPLASTET_PDF = 'OPPLASTET_PDF',
+  MANUELT = 'MANUELT',
```

Comment:
Hva legger du i "manuelt" her? At det er noe saksbehandler har lagt til eller opprettet manuelt, at det er et manuelt/redigerbart brev, eller...?

Reply:
Ja, ser at den er litt utydeleg. Dette seier noko om kva slags type brev det er, og viss det er oppretta frå brev-sida på saka startar dei jo med ein tom mal, så der kan det stå i prinsippet kva som helst. Så det er for dét scenariet.

Reply:
Litt uheldig navn skulle jeg si. Det meste som opprettes fra "Nytt brev" er stort sett informasjonsbrev. 

Ville fjernet MANUELT og heller byttet ut med en type som er mer beskrivende. Dersom det finnes brev som ikke er INFORMASJON, VARSEL, eller OPPLASTET_PDF, er det bedre å legge til en ny type for det.

Reply:
Vi kan jo sjølvsagt leggje til grunn at det er informasjonsbrev, og så satse på at ikkje ein saksbehandlar slumpar til til å bruke nytt brev-funksjonaliteten til å lage andre typar brev.

Vurderte også å bruke namn som VET_IKKE, OPPRETTA_UTEN_MAL, OPPRETTA_DIREKTE_PAA_SAK også. Eller SANNSYNLIGVIS_INFORMASJONSBREV.